frontend/frontend-filters-search.php

```diff
@@ -69,7 +69,7 @@ public function get_search_form( $form ) {
 
 		if ( $this->links_model->using_permalinks ) {
 			// Take care to modify only the url in the <form> tag.
-			preg_match( '#<form.+?>#', $form, $matches );
+			preg_match( '#<form[\s\S]+?>#', $form, $matches );
```

Comment:
FYI, it could also be fixed like this:
```suggestion
			preg_match( '#<form.+?>#s', $form, $matches );
```
Or this:
```suggestion
			preg_match( '#<form.+>#sU', $form, $matches );
```

Reply:
`U`ngreedy not sure is what you want because it matches the whole line until the `</form>`.
`s`ingle line seems sufficient not to process carriage return character. So I chose this one.

Reply:
Done in 64244df1412f71a0a1e3d56bd582207b5da0f61f